src/main/java/TaskManager.java

```diff
@@ -54,4 +54,12 @@ public String returnTaskListAsString() {
         return outputString.toString();
     }
 
+    public static void printTaskList(List<Task> taskList) {
+        int count = 1;
+        System.out.print(UserInterface.line);
+        for (Task task : taskList) {
+            System.out.println("\t" + count++ + ". " + task);
+        }
+        System.out.println(UserInterface.line);
```

Comment:
if line is a static constant in UserInterface maybe can consider naming it LINE instead?